src/libs/shim-services/src/shim_mgmt/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// The shim management client module
+pub mod client;
+
+/// The key for direct volumn path
+pub const DIRECT_VOLUMN_PATH_KEY: &str = "path";
+/// URL for stats direct volumn
+pub const DIRECT_VOLUMN_STATS_URL: &str = "/direct-volumn/stats";
+/// URL for resizing direct volumn
+pub const DIRECT_VOLUMN_RESIZE_URL: &str = "/direct-volumn/resize";
```

Comment:
I know we have this identifier in the repo already but I'm wondering if it's a typo? Should it be...

```suggestion
/// URL for resizing direct volume
pub const DIRECT_VOLUME_RESIZE_URL: &str = "/direct-volume/resize";
```

Reply:
Fixed, nice catch.